src/modules/modals/EditRow.vue

```diff
@@ -190,6 +201,44 @@ export default {
 			this.localLoading = false
 			this.actionCancel()
 		},
+		async loadStore() {
+			const { default: store } = await import(/* webpackChunkName: 'store' */ '../../store/store.js')
```

Comment:
In theory we could also make the store available in the widgets vue instance, but also good as it is as using the full store might also require some bigger changes then to make it aware of a tables app context that is not the full app.